apps/src/templates/instructions/RubricField.jsx

```diff
@@ -65,16 +74,14 @@ class RubricField extends Component {
     return (
       <div style={performanceHeaderStyle}>
         {this.props.showFeedbackInputAreas && (
-          <input
-            type={'checkbox'}
+          <CheckRadioButton
             //Concatenate because injecting the string seems to result in all of
-            //the input elements having the name id
+            //the input elements having the same id
```

Comment:
i'm confused about this comment. why does string interpolation cause each string to have the same id? what is the resulting id?